Lesson26/app/deniz/script.js

```diff
@@ -0,0 +1,52 @@
+// Register on Unsplash to get your own API key
+const API_KEY = '3HO6COGJ_0Qsi2DxSVvlppxQBAHZj4jCPBlARAusM3g';
+
+// TODO: Add a new button in HTML to crear the screen
+
+const button = document.getElementById('fetch-button');
```

Comment:
There are several buttons used in this assignment, better to use a more descriptive name:
```suggestion
const fetchButton = document.getElementById('fetch-button');
```